### PR TITLE
thomasgauvin: add changelog entry for response bodies in bulk operations

### PR DESCRIPTION
**Pull request by thomasgauvin**

### CODE REVIEW

File: src/content/changelogs/kv.yaml [hunk 1]
```diff
@@ -5,6 +5,21 @@ productLink: "/kv/"
...
+      The REST API endpoints for bulk operations ([write](/api/operations/workers-kv-namespace-write-multiple-key-value-pairs), [delete](workers-kv-namespace-delete-multiple-key-value-pairs)) now return the keys of operations that failed during the bulk operation. The updated response bodies are documented in the [REST API documentation](/api/operations/workers-kv-namespace-list-namespaces) and contain the following information:
+
+      ```
+      {
+        "successful_key_count": number,
+        "unsuccessful_keys": string[]
+      }
+      ```
+
+      The unsuccessful keys are an array of the keys that have unsuccessfully been written to all storage backends of Workers KV and should be retried.
```
Comment (rts-rob) [inline src/content/changelogs/kv.yaml:21]: Suggestion: 
"The unsuccessful keys are an array of keys that were not written successfully to all storage backends and should be retried."

Comment (rts-rob) [inline src/content/changelogs/kv.yaml]:
```suggestion
      The unsuccessful keys are an array of keys that were not written successfully to all storage backends and should be retried.
```

File: src/content/changelogs/kv.yaml [hunk 2]
```diff
@@ -5,6 +5,21 @@ productLink: "/kv/"
...
+      The REST API endpoints for bulk operations ([write](/api/operations/workers-kv-namespace-write-multiple-key-value-pairs), [delete](workers-kv-namespace-delete-multiple-key-value-pairs)) now return the keys of operations that failed during the bulk operation. The updated response bodies are documented in the [REST API documentation](/api/operations/workers-kv-namespace-list-namespaces) and contain the following information:
+
+      ```
+      {
+        "successful_key_count": number,
+        "unsuccessful_keys": string[]
+      }
+      ```
+
+      The unsuccessful keys are an array of keys that were not written successfully to all storage backends and should be retried.
```
Comment (Oxyjun) [inline src/content/changelogs/kv.yaml]:
```suggestion
      The unsuccessful keys are an array of keys that were not written successfully to all storage backends and therefore should be retried.
```

**Review by Oxyjun — APPROVED** [review]

File: src/content/changelogs/kv.yaml [hunk 3]
```diff
@@ -5,6 +5,21 @@ productLink: "/kv/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: "2024-11-14"
+    title: Workers KV REST API bulk operations provide granular errors
+    description: |-
+
+      The REST API endpoints for bulk operations ([write](/api/operations/workers-kv-namespace-write-multiple-key-value-pairs), [delete](workers-kv-namespace-delete-multiple-key-value-pairs)) now return the keys of operations that failed during the bulk operation. The updated response bodies are documented in the [REST API documentation](/api/operations/workers-kv-namespace-list-namespaces) and contain the following information:
```
Comment (thomasgauvin) [inline src/content/changelogs/kv.yaml]:
```suggestion
      The REST API endpoints for bulk operations ([write](/api/operations/workers-kv-namespace-write-multiple-key-value-pairs), [delete](workers-kv-namespace-delete-multiple-key-value-pairs)) now return the keys of operations that failed during the bulk operation. The updated response bodies are documented in the [REST API documentation](/api/operations/workers-kv-namespace-list-namespaces) and contain the following information in the `result` field:
```

**Pull request merged**